taas/mapping.py

```diff
@@ -0,0 +1,99 @@
+from abc import ABCMeta, abstractmethod
+
+
+class Mapping(object):
+    """
+        This class handles mappings between our raw input, and what we generate.
+    """
+
+    # We're an abstract class
+    __metaclass__ = ABCMeta
+
+    # This was originally called 'value', but apparently that's a
```

Comment:
This is something I've wrestled with myself, but I've found a convention I've seen in other languages to be useful, that of doing:
```
# NOTE(Jason Ritzke): This is a note
```
Which makes things clear and more easily parsed (and differentiated from TODO, FIXME, etc.